.github/workflows/ci.yml

```diff
@@ -136,50 +206,78 @@ jobs:
         ports:
           - 5432:5432
         # needed because the postgres container does not provide a healthcheck
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        options:
+          --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+          --name postgres
 
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v4
+      - name: Set up backend environment
+        uses: maykinmedia/setup-django-backend@v1.3
         with:
+          apt-packages: 'libxml2 libxmlsec1 libxmlsec1-openssl gettext postgresql-client libgdal-dev gdal-bin'
           python-version: '3.11'
-          cache: 'pip'
-          cache-dependency-path: 'requirements/*.txt'
-      - uses: actions/setup-node@v3
-        with:
-          node-version: '18'
-
-      - name: Install system packages
-        run: |
-          sudo apt-get update \
-          && sudo apt-get install -y --no-install-recommends \
-            libgdal-dev \
-            gdal-bin
+          optimize-postgres: 'yes'
+          pg-service: 'postgres'
+          setup-node: 'yes'
 
       - name: Bring up Alfresco
         run: |
           docker compose -f docker-compose.ci.cmis.yml up -d
           docker ps -q | xargs  docker stats --no-stream
 
-      - name: Install dependencies
-        run: pip install --requirement requirements/ci.txt
-
-      - name: Build frontend
-        run: |
-          npm ci --legacy-peer-deps
-          npm run build --production
-
       - name: Run tests
         if: ${{ !(matrix.cmisurlmapping == 'True' && matrix.binding == 'BROWSER') }}
         run: |
+          echo "# Profiling stats" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_OUTPUT
+
+          python src/manage.py compilemessages
           python src/manage.py collectstatic --noinput --link
-          coverage run src/manage.py test src
+          coverage run src/manage.py test src --tag cmis
```

Comment:
Am I correct that CMIS tests are not run in parallel? Afaik CMIS tests are quite slow, perhaps in the future running them in parallel could also speed up the process

Reply:
yes, I tried running them in parallel, but since they all operate on the same docker container I wasn't able to get it to work. Also since the total time for them to complete is less than the non-CMIS part of the test suite, I figured it wasn't worth my time to try and optimize this